hashtagsv2/hashtags/templates/hashtags/all_projects.html

```diff
@@ -0,0 +1,60 @@
+{% extends "hashtags/base.html" %}
+{% load static %}
+{% load url_replace %}
+
+{% block content %}
+<div class="container">
+    <div class="info">
+        <div class="row">
+            <div class="two-thirds column">
+            <h2>
+                {% for query in hashtag_query_list %}
+                #{{ query }}
+                {% endfor %}
+            </h2>
+            <h2>Projects</h2>
+            </div>
+            <div class="one-third column">
```

Comment:
We should probably make the stats box a template that we include on each of these pages, instead of rewriting it on each page it appears.